CONTRIBUTING.md

```diff
@@ -1,14 +0,0 @@
-
-# Contributing
```

Comment:
Why are we deleting this?

Reply:
I think I may have moved it under a docs folder before and it got lost in the rebase. I will re add it